ui/app/ducks/gas/gas.duck.js

```diff
@@ -161,6 +171,39 @@ async function fetchExternalBasicGasEstimates(dispatch) {
     setStorageItem('BASIC_PRICE_ESTIMATES_LAST_RETRIEVED', timeRetrieved),
   ]);
   dispatch(setBasicPriceEstimatesLastRetrieved(timeRetrieved));
+  return basicEstimates;
+}
+
+async function fetchEthGasPriceEstimates(state) {
+  const chainId = getCurrentChainId(state);
+  const timeLastRetrieved =
```

Comment:
Nit: Not super important, but, it would be slightly faster if we used `Promise.all` to ask for the last retrieved time and the basic price estimates in parallel.